packages/@aws-cdk/aws-docdb/lib/cluster.ts

```diff
@@ -146,6 +146,22 @@ export interface DatabaseClusterProps {
    * @default - false
    */
   readonly deletionProtection?: boolean;
+
+  /**
+   * Should sending profiler logs to CloudWatch Logs should be enabled?
+   * You have to configure the profiler additionally in the parameter group.
+   * @see https://docs.aws.amazon.com/documentdb/latest/developerguide/profiling.html#profiling.enable-profiling
+   * @default false
+   */
+  readonly exportProfilerLogsToCloudWatch?: boolean;
```

Comment:
Please incorporate this comment: https://github.com/aws/aws-cdk/pull/17570#discussion_r752788270.